sale_financial_risk/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,7 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Carlos Dauden
+  * Pedro M. Baeza
+  * Ernesto Tejeda
+
+* Agathe Mollé <agathe.molle@savoirfairelinux.com>
```

Comment:
If this will be contributed back to OCA, perhaps you ought to include yourself as contributor?